pkg/codegen/dotnet/gen.go

```diff
@@ -2198,6 +2198,12 @@ func genProjectFile(pkg *schema.Package,
 	packageReferences map[string]string,
 	projectReferences []string) ([]byte, error) {
 
+	if packageReferences == nil {
+		packageReferences = map[string]string{}
+	}
+	if _, ok := packageReferences["Pulumi"]; !ok {
+		packageReferences["Pulumi"] = "3.*"
```

Comment:
Is this reference accurate - is the latest code that gets generated compatible with Pulumi 3.0.0 or should this be something like `[3.2.1,4)`?

Reply:
@iwahbe sorry I wasn't able to review before this got merged ... might require a little more investigation and a follow-up PR

Reply:
No problem. We have `3.*` everywhere. Checking against our examples, we compile against `3.23`.

Reply:
Yeah, this is known to be wrong everywhere currently, but maintaining accurate min versions in each repo right now would be too much work. Now this is centralised, we should be bumping the min version as we add features to the SDK for use in codegen. We should do a one-off check if there have been any features added in the .Net SDK which are required for the current version of codegen.